spec/controllers/pdfs_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe PdfsController, type: :controller do
+  it "should respond back with a pdf" do
+    allow_any_instance_of(Paperclip::Attachment).to receive(:url).and_return("spec/fixtures/sample_pdf.pdf")
```

Comment:
stubbing the paperclip url here so it points to fixtures, not the default url that I assigned for paperclip.